examples/plotting/solar_cycle_example.py

```diff
@@ -13,14 +14,18 @@
 from sunpy.time import parse_time
 
 ###############################################################################
-# For this example we will use the SunPy sample data. This code snippet grabs
-# the most current NOAA solar cycle data as a ``TimeSeries``.
+# For this example we will use sunpy's sample data which has an old
+# sunspot record from 2015 and prediction for the cycle till 2019 from (SWPC)
+# part of the National Oceanic and Atmospheric Administration (NOAA).
+# This code snippet grabs the NOAA solar cycle data as a ``TimeSeries``.
 noaa = ts.TimeSeries(NOAAINDICES_TIMESERIES, source='noaaindices')
```

Comment:
To download the data from the server you should just use
```python
from sunpy.net import Fido, attrs as a
result = Fido.search(a.Time("1990/1/1", datetime.datetime.now()), a.Instrument('noaa-indices'))
f = Fido.fetch(result)
result = Fido.search(a.Time(<today>, datetime.datetime.now()+ datetime.timedelta(days=4*365)), a.Instrument(''noaa-predict''))
f_predict = Fido.fetch(result)```
The dates may have to be messed around with. You can then load the files into the TimeSeries as before.

Reply:
Hi, I've updated the PR to download the data and use it as time-series. Thanks a lot for the Fido snippet for downloading data.